frontend/lib/tracking/ga.ts

```diff
@@ -0,0 +1,78 @@
+interface TrackerConfig {
+    name: string;
+    id: string;
+    sampleRate: number;
+}
+interface Config {
+    tracker: TrackerConfig;
+    pillar: Pillar;
+}
+
+// const getAnalyticsEdition = () => {
+//     const edition = (guardian.config.page.edition || '').toLowerCase();
+//     return edition === 'int' ? 'international' : edition;
+// };
+
+export const init: (_: Config) => void = ({ tracker, pillar }) => {
+    const coldQueue = (...args: any[]) => {
+        (ga.q = ga.q || []).push(args);
+    };
+
+    ga = ga || (coldQueue as UniversalAnalytics.ga);
+    ga.l = +new Date();
+    ga('create', tracker.id, 'auto', '@tracker.trackerName', {
+        sampleRate: 100, // WARNING: we are sampling everyone!
+        siteSpeedSampleRate: tracker.sampleRate,
+        // TODO: 'userId': identityId
+    });
+    ga('set', 'forceSSL', true);
+    // ga('set', 'title', guardian.config.page.webTitle || '');
+    ga('set', 'anonymizeIp', true);
+    /***************************************************************************************
+     * Custom dimensions common to all platforms across the whole Guardian estate           *
+     ***************************************************************************************/
+    // ga('set', 'dimension1', guardian.config.ophan.pageViewId);
```

Comment:
This is set by Ophan and should be available by the time this script executes. If not, we'll have to define it inline blocking JS [similar to how frontend does it](https://github.com/guardian/frontend/blob/master/common/app/templates/inlineJS/blocking/config.scala.js#L38-L45)